src/js/util.js

```diff
@@ -18,4 +18,14 @@ export function buildElementClasses (block, elements) {
 	});
 
 	return bemClasses;
+}
+
+export function makeGuid() {
```

Comment:
I made a different but similar method. It might be misleading to call this guid because it has no guarantee of uniqueness, outside of the probabilities you impose by having enough chars.